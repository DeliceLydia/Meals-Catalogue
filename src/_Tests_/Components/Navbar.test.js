import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../Containers/Navbar';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>
  );
  const tree = renderer
    .create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
