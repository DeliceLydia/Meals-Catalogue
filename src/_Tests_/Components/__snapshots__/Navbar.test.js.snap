// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="nav"
>
  <a
    className="home"
    href="/"
    onClick={[Function]}
  >
    <h4
      className="nav-text"
    >
      Delicious Meals
    </h4>
  </a>
</nav>
`;
